src/media/headset.ts

```diff
@@ -0,0 +1,71 @@
+import { Observable } from 'rxjs';
+import { SdkMedia } from "..";
+import HeadsetService, { ConsumedHeadsetEvents, VendorImplementation} from 'softphone-vendor-headsets';
+
+export class SdkHeadset {
+    private media: SdkMedia;
+    private headsetLibrary: HeadsetService;
+    headsetEvents: Observable<ConsumedHeadsetEvents>;
```

Comment:
Little bit of a NIT here, but it is a fairly common pattern when exposing observable streams to postfix the variable name with a `$` (like you have in the headset lib currently). Not a deal breaker.